package.json

```diff
@@ -75,6 +75,7 @@
     "redux-mock-store": "^1.5.3",
     "redux-promise-middleware": "^5.1.1",
     "redux-thunk": "^2.3.0",
+    "react-select": "^1.2.1",
```

Comment:
I don't think we want this here. You sill break all selects in react-ui-components with it.